File: Day1/solution_part2.rb
#!/usr/bin/env ruby

f = File.read("input.txt")
comp_indx = f.length/2
sum = 0
compare = f[comp_indx]
f.each_char do |s|
    if compare.eql? s
        sum += s.to_i
    end

    comp_indx += 1
    if comp_indx >= f.length
        comp_indx = comp_indx - f.length
        compare = f[comp_indx]
    else
        compare = f[comp_indx]
    end
end

puts "TOTAL SUM: "+sum.to_s

File: Day2/solution_part2.py
#!/usr/bin/env python

with open('input.txt') as f:
    content = [x.split("\t") for x in [x.strip('\n') for x in f.readlines()]]

divs = []
for x in content:
    for i in x:
        for j in x:
            if (int(i)%int(j) == 0 and i != j):
                divs.append(int(i)/int(j))

print sum(divs)